pyproject.toml

```diff
@@ -22,28 +22,24 @@ classifiers = [
 requires-python = ">= 3.9"
 dependencies = ["pytest>=7.0.0"]
 
+[project.optional-dependencies]
+dev = [
+    "coverage[toml] >= 7.9",
+    "pytest-httpbin >= 2.1.0",
+    "pytest-randomly >= 3.15.0",
+    "requests >= 2.32.4",
+    "starlette >= 0.47.1",
+    "httpx >= 0.28.1",
+]
+
 [project.urls]
 "Homepage" = "https://pypi.org/project/pytest-socket/"
 "Repository" = "https://github.com/miketheman/pytest-socket"
 "Bug Tracker" = "https://github.com/miketheman/pytest-socket/issues"
 "Change Log" = "https://github.com/miketheman/pytest-socket/blob/main/CHANGELOG.md"
 "Funding" = "https://github.com/sponsors/miketheman"
 
-[tool.poetry]
-include = [
-    { path = "LICENSE", format = "sdist" },
-    { path = "README.md", format = "sdist" },
-    { path = "tests", format = "sdist" },
-    { path = ".flake8", format = "sdist" },
-]
```

Comment:
This was removed from the outcome. We'd like to have these files included in the source distribution (sdist) file, it may need some sdist-specific configuration for uv build backend.